manual/Makefile

```diff
@@ -1,9 +1,38 @@
 # Makefile for the reference and user manuals
 TEXDIR=tex_files
 VPATH=$(TEXDIR)
-.PHONY:all
+.PHONY:all clean sphinx html pdf old doxygen
 
-all: user_manual.pdf \
+# some distros may provide sphinx-build-3 and sphinx-build-2 instead
+# of sphinx-build. An automatic check would be helpfull, but at least
+# with this, it should be easy to change:
+sphinx-build=sphinx-build
+
+all: sphinx
+manual: all
+# set some shorter names
+pdf: sphinx-pdf
+html: sphinx-html
+sphinx: sphinx-pdf
+
+sphinx-pdf:
```

Comment:
Missing dependency on `breathe-autogen`

Reply:
This is now added, but failure doesn't stop the build (uses '-' before shell command)